data/80-udisks2.rules

```diff
@@ -126,6 +126,11 @@ ENV{ID_PART_ENTRY_SCHEME}=="gpt", \
   ENV{ID_PART_ENTRY_TYPE}=="c12a7328-f81f-11d2-ba4b-00a0c93ec93b|21686148-6449-6e6f-744e-656564454649|a19d880f-05fc-4d3b-a006-743f0f84911e|e6d6d379-f507-44c2-a23c-238f2a3df928|e3c9e316-0b5c-4db8-817d-f92df00215ae|de94bba4-06d1-4d40-a16a-bfd50179d6ac", \
   ENV{UDISKS_IGNORE}="1"
 
+# ZFS member partitions
+ENV{ID_PART_ENTRY_SCHEME}=="gpt", \
```

Comment:
> ZFS on MBR partitions/slices is extremely rare, this was to avoid unnecessary scanning.

This is not about unnecessary scanning. The udev rule only compares a few properties that are already set anyway. So the real question is if the above combination is the only one it should match. And it looks like it isn't. What if somebody for example has an MBR-partitioned SSD and wants to use one of the partitions as a cache for ZFS?